apps/coscrad-frontend/src/coscrad-theme-provider.tsx

```diff
@@ -17,6 +45,17 @@ export const CoscradThemeProvider = ({ children }: CoscradThemeProviderProps): J
                     main: blue[800],
                 },
             },
+            typography: {
+                small: {
```

Comment:
If we decide to break out our own series of sizes based on our context, we should come up with a standard set that is clear to other developers.  @aaron-plahn for some reason `responsiveFontSizes()` doesn't recognize the additional custom typography variants I've added.  Any idea why?  I've sleuthed a bit through @mui looking for some insight but I can't figure out how the types all flow.  I see that `Variant` is exported with an alias: `Variant as TypographyVariant`

Reply:
Can we discuss this on a call? I need some context.